### PR TITLE
Decode spirc frame protobuf

### PR DESCRIPTION
**Pull request by kingosticks**
It's interesting (and maybe useful too) to see what's going on in those Frame messages. Something similar could be added for the other message types too. 

Sometimes there isn't a URI and I'm not sure why, maybe it's linked to the packets with the bad lengths.

I also added something to specifically highlight which were audio key messages, but only because I happen to look into what they were.

### CODE REVIEW

File: dissect/dissect.sh [hunk 1]
```diff
@@ -8,4 +8,5 @@ exec wireshark-gtk $1 \
     -X lua_script:gaia.lua \
     -X lua_script1:proto/keyexchange.proto \
     -X lua_script1:proto/authentication.proto \
+    -X lua_script1:proto/spirc.proto \
```
Comment (devgianlu) [inline dissect/dissect.sh:11]: This needs to be added to `dissect.cmd` too.

**Comment by devgianlu** [comment]
Works flawlessly for me. 

Is it possible to decode `context_player_state` so that you don't have to copy-paste it somewhere else? It is gzipped.

**Comment by kingosticks** [comment]
It didn't look like wireshark has a builtin gzip dissector, I did try to do exactly this myself.

**Comment by devgianlu** [comment]
It appears `zlib` isn't very different from `gzip`. If you can use an external library [zlib](https://github.com/brimworks/lua-zlib) should work.

**Comment by sashahilton00** [comment]
From a quick google, oen can decompress it via the GUI. Going to merge for now, feel free to add zlib stuff later, but I'm not sure it's necessary.

**Comment by kingosticks** [comment]
Can you post the link to how do that in the gui? All the "decode as" options were greyed out for me in my version of wireshark and I assumed that was because it was already decoding using the probuf decoder.

**Comment by sashahilton00** [comment]
https://stackoverflow.com/questions/12925769/wireshark-doesnt-decode-gziped-http-traffic-on-windows looked about right. I've definitely seen the `Export packet bytes as` option, along with the `Show packet bytes` option, though I'm not sure the latter allows gzip decompression.

One might also be able to do something with this: https://github.com/kizzx2/wireshark-http-gunzip

**Comment by kingosticks** [comment]
It doesn't since it has no idea how to interpretate those byes from a totally unknown protocol. I'm going to look into calling something external from the lua. The other option sounds clunky.

**Comment by sashahilton00** [comment]
really? You can't even export the packet bytes and force it to try gunzipping them?

**Comment by kingosticks** [comment]
I believe it was greyed out within wireshark.  maybe you can still just export the raw bytes to a file (I don't remember) and then manually gunzip yourself but that's very clunky. I'll try again next week if nobody beats me to it.

**Pull request merged**